cypress/integration/with-security/check_advanced_settings.js

```diff
@@ -25,13 +25,6 @@ describe('verify the advanced settings are saved', () => {
       .should('eq', 'true');
   });
 
-  it('the Timeline default columns field is set to 4', () => {
-    cy.get('[data-test-subj="advancedSetting-editField-timeline:default_columns"]').should(
-      'have.value',
-      4
-    );
-  });
-
   it('the Timeline Maximum buckets field is set to 4', () => {
```

Comment:
Should this also be deleted? Or is it used by `vis_type_timeline`?

Reply:
`timeline:default_columns` is a specific setting for the timeline application, used to determine the number of default columns shown in the timeline view. In timeline visualization, we don't have such settings.

Reply:
got it, so this was a setting just for timeline sheets.